Raygun4UWP.nuspec

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata minClientVersion="2.8.3">
+    <id>Raygun4UWP</id>
+    <title>Raygun4UWP</title>
+    <version>1.0.0</version>
+    <releaseNotes>
+    </releaseNotes>
+    <authors>Raygun</authors>
+    <copyright>Copyright 2019</copyright>

```

Comment:
Should probably be `Copyright © 2019 Raygun`

Reply:
Changed to "Copyright © Raygun 2019" to be consistent with the AssemblyInfo.cs of this and other providers.